### PR TITLE
Making all chainsaws fragile

### PR DESCRIPTION
**Pull request by guralub**
#### Summary

SUMMARY: Balance "Add the FRAGILE flag to all variants of the chainsaw to better balance them."

#### Purpose of change

Better balance chainsaw as weapons, as per the suggestion of mlangsdorf in the Discord server.

#### Testing

Tested to see if the changes worked as intended

### CODE REVIEW

File: data/json/items/melee/swords_and_blades.json [hunk 1]
```diff
@@ -1814,6 +1814,6 @@
     "revert_to": "e_combatsaw_off",
     "qualities": [ [ "AXE", 3 ] ],
     "use_action": [ "E_COMBATSAW_ON" ],
-    "flags": [ "MESSY", "DURABLE_MELEE", "TRADER_AVOID", "POWERED", "ALWAYS_TWOHAND" ]
+    "flags": [ "MESSY", "FRAGILE_MELEE", "TRADER_AVOID", "POWERED", "ALWAYS_TWOHAND" ]
```
Comment (Leland) [inline data/json/items/melee/swords_and_blades.json:1817]: Should an "electric combat chainsaw" be fragile? The description notes that it's been "extensively modified to be a more effective weapon"

Comment (anothersimulacrum) [reply data/json/items/melee/swords_and_blades.json:1817]: image

Comment (Leland) [reply data/json/items/melee/swords_and_blades.json:1817]: Per our in-game note:

https://github.com/CleverRaven/Cataclysm-DDA/blob/4d8d304761939350dd253841da4aea67f7eaa296/data/json/flags.json#L317

An item named a "combat chainsaw," while being described to a user as being unsuited for combat, is a bit confusing!

Also note our internal [markdown docs](https://github.com/CleverRaven/Cataclysm-DDA/blob/85b5249d291ee2e6503ca3fb8f6f7226b7b3ce0a/doc/JSON_FLAGS.md#L679):

> Fragile items that fall apart easily when used as a weapon due to poor construction quality and will break into components when broken.

I wouldn't imagine an "effective weapon" as also suffering from poor construction quality. Perhaps we need to update our documentation?

Comment (guralub) [reply data/json/items/melee/swords_and_blades.json:1817]: I guess these statements do contradict one another. Would changing the description notes to "extensively modified to be easier to handle as a weapon" be any good?

**Pull request merged**